history-of-the-quakers/original/29-book-3-chapter-3.adoc

```diff
@@ -401,86 +428,89 @@ he had openly despised and treated with disdain, and which, as foretold,
 overtook him as a thief in the night.]
 By what law will you put me to death?
 
-Court.
+_Court._
 We have a law, and by our law, you are to die.
 
-Wenlock.
+_Wenlock._
 So said the Jews of Christ, We have a law, and by our law he ought to die:
 Who empowered you to make that law?
 
-Court.
+_Court._
 We have a patent and are patentees, judge whether we have not power to make laws?
 
-Wenlock.
+_Wenlock._
 How! have you power to make laws repugnant to the laws of England?
 
-Governor.
+_Governor._
 Nay.
 
-Wenlock.
+_Wenlock._
 Then you are gone beyond your bounds, and have forfeited your patent,
 and this is more than you can answer.
 Are you subjects to the King, yea or nay?
 
-Secretary Rawson.
+_Secretary Rawson._
 What will you infer from that, what good will that do you?
 
-Wenlock.
+_Wenlock._
 If you are, say so; for in your petition to the King,
 you desire that he will protect you,
 and that you may be worthy to kneel among his loyal subjects.
 
-Court.
+_Court._
 Yes.
 
-Wenlock.
+_Wenlock._
 So am I, and for any thing I know am as good as you, if not better:
 for if the King did but know your hearts, as God knows them,
 he would see that your hearts are as rotten towards him as they are towards God.
 Therefore seeing that you and I are subjects to the King,
 I demand to be tried by the laws of my own nation.
 
-Court.
+_Court._
 You shall be tried by a bench and jury.^
 footnote:[Those who had been condemned to death before him were deprived of this privilege;
 but the government of England being changed by the King`'s restoration,
 they began to be afraid to go on in the former course,
 of condemning without a trial by jury,
 as being subversive of the fundamental laws of England, and rights of Englishman.]
 
-Wen lock.
+_Wenlock._
```

Comment:
i won't leave a comment on all of them, but all the chunks of this discourse, i think should be discourse-part